docker/ci/github/Dockerfile

```diff
@@ -14,11 +14,11 @@ ENV CARGO_HOME "/opt/cargo/"
 RUN mkdir -p /github/home \
     && mkdir -p /opt/cargo/ \
     && mkdir -p /opt/git/ \
-    && /diem/scripts/dev_setup.sh -t -o -b -p -y \
+    && /diem/scripts/dev_setup.sh -t -o -b -p -y -s\
```

Comment:
hmmm I think this might break on mac, but not sure, a force push to gha-test-1 would let us know if it'll break the sccache stuff.

Reply:
sure I can push to gha-test-1

Reply:
I checked that the package names make sense on mac but I don't know how to test on mac

Reply:
git push -f upstream <yourbranch>:gha-test-1 will trigger the sccache populating job that runs on mac os

Reply:
It's running here: https://github.com/diem/diem/actions/runs/582996987

Reply:
Looks good, only warning that might trip up things, but otherwise all good.

Reply:
https://github.com/diem/diem/runs/1939308877?check_suite_focus=true#step:6:962

Reply:
Nice